Nyakaru/Bitcoin-Challenge
# Add dashboard and its pages

**What does this PR do?** 
Add home and wishlist views
**Description of Task to be completed?** 
Add coindesk apis and their supporting views
 **How should this be manually tested?** 
- Create a local copy `git fetch origin ch-dashboard` 
- Checkout to the branch `git checkout ch-dashboard` 
-  Cd into client directory by running `cd client`
- Install project dependencies run `yarn` 
- Run the dev server `yarn start` 
- You should be to access dashboard on the browser
